corehq/apps/users/templates/users/web_users.html

```diff
@@ -312,6 +312,12 @@ <h3 class="panel-title">
                                 data-content="{% trans "Create, update, and delete Locations in the Organization Hierarchy." %}"></span>
                         </th>
                         <th>
+                          {% trans "Edit MOTECH Configuration" %}
+                          <span class="hq-help-template"
+                                data-title="{% trans 'Edit MOTECH Configuration' %}"
+                                data-content="{% trans 'Manage integration with other systems.' %}"></span>
+                        </th>
```

Comment:
Can you make this row only show up if you have the openmrs flag set? We shouldn't expose a checkbox that does nothing (whether checked or unchecked) for the vast majority of users.